stdlib/public/core/Integers.swift.gyb

```diff
@@ -3132,6 +3158,14 @@ extension UnsignedInteger where Self : FixedWidthInteger {
   public static var min: Self {
     return 0
   }
+
+  @inlinable
+  public func isMultiple(of other: Self) -> Bool {
+    // Nothing but zero is a multiple of zero.
+    if other == 0 { return self == 0 }
```

Comment:
Why not use of `guard` for an early return?

Reply:
In this case I think it reads a lot more clearly as an `if`, because it avoids the double-negative.

Reply:
Make sense! @stephentyrone thanks for your response 😉